### PR TITLE
PAYARA-2176 Fixed ordering issue between tests

### PR DESCRIPTION
**Pull request by arjantijms**
PAYARA-2176 Added ordering directive for surefire and renamed classed to fit

### CODE REVIEW

**Comment by arjantijms** [comment]
jenkins quick test payara 5 please

**Comment by arjantijms** [comment]
jenkins quick test payara 5 please

**Comment by arjantijms** [comment]
jenkins quick test payara 5 please

File: nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextManagerImplUnitTest.java [hunk 1]
```diff
@@ -77,7 +77,8 @@ public void registerContextFactoryForPrefixNamed(String prefixName, ContextViewF
 			}
 		};
 
-		ContextManagerImpl cmi = new ContextManagerImpl();
+		@SuppressWarnings("unused")
+        ContextManagerImpl cmi = new ContextManagerImpl();
```
Comment (Pandrex247) [inline nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextManagerImplUnitTest.java]: Indentation

File: nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/AContextImplContextPropagationIntegrationTest.java [hunk 2]
```diff
@@ -91,16 +91,17 @@ public void setUpOncePerTestMethod() {
 	public void testThreadLocalBehaviour() {
 		Context diagnosticsContextStart = mContextManager.getContext();
 
-		assertEquals("The implementation class of diagnosticsContext1 is not as expected.",
-		        diagnosticsContextStart.getClass().getName(), ContextImpl.class.getName());
+		assertEquals(
+	        "The implementation class of diagnosticsContext1 is not as expected.",
```
Comment (Pandrex247) [inline nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/AContextImplContextPropagationIntegrationTest.java]: When dropping down it needs two indents.

File: nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/AContextImplContextPropagationIntegrationTest.java [hunk 3]
```diff
@@ -91,16 +91,17 @@ public void setUpOncePerTestMethod() {
...
+		    diagnosticsContextStart.getClass().getName(), ContextImpl.class.getName());
```
Comment (Pandrex247) [inline nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/AContextImplContextPropagationIntegrationTest.java]: Should be aligned with previous line.

File: nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/AContextImplContextPropagationIntegrationTest.java [hunk 4]
```diff
@@ -125,10 +126,12 @@ public void testValuePersistence() {
 		for (int i = 0; i < 17; i++) {
 			Context diagnosticsContext = mContextManager.getContext();
 
-			Assert.assertEquals("The value associated with key " + propagatingKey + " is not as expected.", propagatingValue,
-			        diagnosticsContext.get(propagatingKey));
-			Assert.assertEquals("The value associated with key " + nonPropagatingKey + " is not as expected.", nonPropagatingValue,
-			        diagnosticsContext.get(nonPropagatingKey));
+			assertEquals(
+		        "The value associated with key " + propagatingKey + " is not as expected.",
```
Comment (Pandrex247) [inline nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/AContextImplContextPropagationIntegrationTest.java]: Needs a double indent.

File: nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/AContextImplContextPropagationIntegrationTest.java [hunk 5]
```diff
@@ -153,17 +156,20 @@ public void testValuePropagationAndNonPropagation() throws Exception {
...
 					try {
 						String threadName = currentThread().getName();
 						Context diagnosticsContext = contextManager.getContext();
 
 						assertEquals(
-						        "The value associated with key " + propagatingKey + " on thread " + threadName + " is not as expected.",
-						        propagatingValue, diagnosticsContext.get(propagatingKey));
-						assertNull("The null value should be associated with key " + nonPropagatingKey + " on thread " + threadName,
-						        diagnosticsContext.get(nonPropagatingKey));
+					        "The value associated with key " + propagatingKey +
```
Comment (Pandrex247) [inline nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/AContextImplContextPropagationIntegrationTest.java]: Needs a double indent.

File: nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/AContextImplContextPropagationIntegrationTest.java [hunk 6]
```diff
@@ -153,17 +156,20 @@ public void testValuePropagationAndNonPropagation() throws Exception {
...
-						        propagatingValue, diagnosticsContext.get(propagatingKey));
-						assertNull("The null value should be associated with key " + nonPropagatingKey + " on thread " + threadName,
-						        diagnosticsContext.get(nonPropagatingKey));
+					        "The value associated with key " + propagatingKey + 
+					        " on thread " + threadName + " is not as expected.",
+					        propagatingValue, diagnosticsContext.get(propagatingKey));
+						assertNull(
+					        "The null value should be associated with key " + nonPropagatingKey + 
+					        " on thread " + threadName,
+						    diagnosticsContext.get(nonPropagatingKey));
```
Comment (Pandrex247) [inline nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/AContextImplContextPropagationIntegrationTest.java]: Should be aligned with previous line, which should be double indented.

File: nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextManagerImplUnitTest.java [hunk 7]
```diff
@@ -77,7 +77,8 @@ public void registerContextFactoryForPrefixNamed(String prefixName, ContextViewF
 			}
 		};
 
-		ContextManagerImpl cmi = new ContextManagerImpl();
+		@SuppressWarnings("unused")
```
Comment (michaelranaldo) [inline nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextManagerImplUnitTest.java]: Is this correct? It appears to be briefly used on line 116

File: nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextImplUnitTest.java [hunk 8]
```diff
@@ -170,7 +170,9 @@ public void testDelegationOfGet(@Mocked final Location mockedLocation, @Mocked f
 
 		String value = contextImpl.get(key);
 
-		Assert.assertEquals("Value returned from contextImpl.get(\"" + key + "\") is not the value expected.", expectedValueOfKey1, value);
+		assertEquals(
+	        "Value returned from contextImpl.get(\"" + key + "\") is not the value expected.",
```
Comment (Pandrex247) [inline nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextImplUnitTest.java]: Needs a double indent

File: nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextManagerImplUnitTest.java (hunk 7 above)
Comment (arjantijms) [reply nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextManagerImplUnitTest.java]: I'm not sure, it's in the original test, and potentially it tests whether the instance can be created without throwing exceptions.

File: nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextManagerImplUnitTest.java (hunk 1 above)
Comment (arjantijms) [reply nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextManagerImplUnitTest.java]: It's indented allright (seemingly, on my screen), but it looks like the original code used tabs. I'll change them all to spaces

**Comment by arjantijms** [comment]
jenkins quick test payara 5 please

**Pull request merged**